excel2sbol/excel2sbol/comp_column_functions.py

```diff
@@ -416,6 +416,22 @@ def objectType(self):
 
     def types(self):
         # used to decide the molecule type in the converter function
+        cell = []
+
+        # Assuming that circular will come after molecule type column in most templates
+        if self.cell_val[0:4] == 'http':
+            if getattr(self.obj, self.sbol_term_suf)[0][0:4] == 'http':
+                cell.append(getattr(self.obj, self.sbol_term_suf)[0])
```

Comment:
I believe this still undoes the fix for Sai's sheet. You should ask here about DNA versus RNA and protein types and run her sheet through as a test.

Reply:
Which one is her sheet? 
Also, when you speak about DNA versus RNA, is that in saying unless the type is labelled as RNA or protein, it should be DNA, and then circular has its own case within the funciton?

Reply:
https://github.com/SynBioDex/Excel-to-SBOL/blob/master/excel2sbol/tests/test_files/test_version7_flapjack_compiler_sbol3_v0025.xlsx but you can slack her for the most up to date one

Reply:
https://github.com/SynBioDex/Excel-to-SBOL/blob/master/excel2sbol/tests/test_files/test_version7_flapjack_compiler_sbol3_v0025.xlsx but you can slack her for the most up to date one

Reply:
Thanks!